spring-cloud-azure-eventhub-stream-binder/src/main/java/com/microsoft/azure/eventhub/stream/binder/EventHubMessageChannelBinder.java

```diff
@@ -92,6 +92,16 @@ public EventHubProducerProperties getExtendedProducerProperties(String destinati
         return this.bindingProperties.getExtendedProducerProperties(destination);
     }
 
+    @Override
+    public String getDefaultsPrefix() {
```

Comment:
Why there is an `s` in `Defaults`?

Reply:
This `defaults` is define by spring cloud stream super class. As I understand, the meaning of plural is that there're many default property values.